templates/service/app/config/Module.scala

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmrc.$!APP_PACKAGE_NAME!$.config
+
+import com.google.inject.AbstractModule
+
+class Module extends AbstractModule {
+
+  override def configure(): Unit = {
+
+    bind(classOf[AppConfig]).asEagerSingleton()
```

Comment:
RED: I don't think we should be encouraging adding eager singletons just to make tests pass.  If a service does not exercise this, it would be an indication that it is not needed and the better option would be to remove the AppConfig.  I wouldn't expect the init-service to produce 100% code coverage out of the box.